phishtray/base.py

```diff
@@ -76,3 +79,25 @@ def save(self, *args, **kwargs):
     def hard_delete(self):
         super().hard_delete()
         flush_cache()
+
+
+class MultiTenantMixin(models.Model):
+    organization = models.ForeignKey(
+        "participant.Organization", on_delete=models.PROTECT, null=True, blank=True
+    )
+    objects = OrganizationManager()
+
+    class Meta:
+        abstract = True
+
+
+class OrganizationAdminMethods(admin.ModelAdmin):
```

Comment:
Nice base class but the name doesn't give away much as it is, I'd suggest to rename this to `MultiTenantModelAdmin`.